src/Altinn.Platform/Altinn.Platform.Receipt/Receipt/Program.cs

```diff
@@ -35,9 +48,87 @@ public static void Main(string[] args)
                                    .CreateLogger();
                    logging.AddProvider(new SerilogLoggerProvider(logger));
                })
-          .ConfigureWebHostDefaults(webBuilder =>
-          {
-              webBuilder.UseStartup<Startup>();
-          });
+              .ConfigureWebHostDefaults(webBuilder =>
+              {
+                    webBuilder.ConfigureAppConfiguration((hostingContext, config) =>
+                      {
+                          _logger.Information($"Program // ConfigureAppConfiguration");
+
+                          string basePath = Directory.GetParent(Directory.GetCurrentDirectory()).FullName;
+
+                          string basePathCurrentDirectory = Directory.GetCurrentDirectory();
+                          _logger.Information($"Current directory is: {basePathCurrentDirectory}");
+
+                          LoadConfigurationSettings(config, basePath, args);
+                      })
+
+                    .UseStartup<Startup>();
+              });
+
+        /// <summary>
+        /// Load the configuration settings for the program.
+        /// </summary>
+        /// <param name="config">the config</param>
+        /// <param name="basePath">the base path to look for application settings files</param>
+        /// <param name="args">programs arguments</param>
+        public static void LoadConfigurationSettings(IConfigurationBuilder config, string basePath, string[] args)
+        {
+            _logger.Information($"Program // Loading Configuration from basePath={basePath}");
+
+            config.SetBasePath(basePath);
+            string configJsonFile1 = $"{basePath}/altinn-appsettings/altinn-dbsettings-secret.json";
+            string configJsonFile2 = $"{basePath}/Receipt/appsettings.json";
+
+            if (basePath == "/")
+            {
+                configJsonFile2 = "/app/appsettings.json";
+            }
+
+            _logger.Information($"Loading configuration file: '{configJsonFile1}'");
+            config.AddJsonFile(configJsonFile1, optional: true, reloadOnChange: true);
+
+            _logger.Information($"Loading configuration file2: '{configJsonFile2}'");
+            config.AddJsonFile(configJsonFile2, optional: false, reloadOnChange: true);
+
+            config.AddEnvironmentVariables();
+            config.AddCommandLine(args);
+
+            ConnectToKeyVaultAndSetApplicationInsights(config);
```

Comment:
Maybe move this up above AddCommandLine so that any command line arguments takes precedence.

Reply:
ok